web/src/pages/index.tsx

```diff
@@ -17,9 +19,20 @@ export const getServerSideProps: GetServerSideProps = async ({ req, res }) => {
     };
   }
 
+  const team_id = await getFirstTeamIdByUserId(user_id);
+
+  if (!team_id) {
+    return {
```

Comment:
let's make a note this is temporary while we add support for creating a new team or deleting your account

Reply:
Done